src/types.ts

```diff
@@ -36,6 +36,11 @@ export type DraggableFlatListProps<T> = Modify<
     simultaneousHandlers?: React.Ref<any> | React.Ref<any>[];
     outerScrollOffset?: Animated.Node<number>;
     onAnimValInit?: (animVals: ReturnType<typeof useAnimatedValues>) => void;
+    /**
+     * Whether the renderItem prop passed in doesn't need index to passed into
+     * the component. Can be used to improve performance. Defaults to true
```

Comment:
I think this comment could be phrased a bit more specifically to talk about how it improves performance, and to remove the negative in "whether".

"Defaults to true. If set to false, `renderItem` won't be passed the index of the item. This improves render performance by saving a re-render when items' indexes change."